package.json

```diff
@@ -1,23 +1,23 @@
 {
   "name": "@sap-webide/webide-client-tools",
-  "version": "3.1.2",
+  "version": "3.1.3",
```

Comment:
don't bump version manually, according to the docs the bumping is automated
- https://github.com/SAP/webide-client-tools/blob/master/CONTRIBUTING.md#releasing-new-version

Reply:
reverted